client/src/main.js

```diff
@@ -24,7 +24,10 @@ girder.girder.$refresh().then(() => {
   Vue.use(ResonantGeo, {
     girder: girder.girder,
   });
-  Vue.use(VueCesium);
+  Vue.use(VueCesium, {
+    // default Cesium Ion API key
+    accessToken: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI1MDA2NzVhYi05NjljLTQ2ZWQtYWJkZC1jYTg4NzA4YjEzNGMiLCJpZCI6MjU5LCJpYXQiOjE2MjUxNTA1ODh9.hDxkoBgpk13oYPDuKYhkt6XSyii_kUKufDE_GLE38is'
```

Comment:
Can you link to where this was provided?

Reply:
https://github.com/CesiumGS/cesium/blob/main/Source/Core/Ion.js#L6-L7